.pre-commit-config.yaml

```diff
@@ -5,28 +5,25 @@ repos:
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
-      - id: fix-encoding-pragma
-        args: [--remove]
-      - id: check-yaml
-        exclude: ^.+(/tests/|/recipe/).+$
-      - id: check-toml
-      - id: check-json
       - id: check-merge-conflict
-      - id: pretty-format-json
-        args: [--autofix]
       - id: debug-statements
         language_version: python3
+  # Autoformat: YAML, JSON, Markdown, etc.
+  - repo: https://github.com/rbubley/mirrors-prettier
```

Comment:
Using a fork, since official prettier pre-commit hook has been archived:
http://github.com/prettier/prettier/issues/15742

Reply:
Hum, I'm kind of reluctant here. As I'm in favor of general improvements, using something that just has been archived (even if a mirror is available) doesn't seem a good idea to me. 
`https://github.com/rbubley/mirrors-prettier` seems to rather be a workaround to let people keep using the hook and there are no guarantees that it will be maintained in the long term.
I would highly suggest to wait until a stable solution is available (could be something used by a significant number of people, regularly maintained, or officially superseding the archived one in `pre-commit`).

Reply:
There is an important gotcha: the https://github.com/rbubley/mirrors-prettier is not a mirror/fork of `prettier` ore `pre-commit`.
It's a small repo to make vanilla `prettier` work as `pre-commit` hook.
If you take a look at the source code, you'll see that the part which matters is less than 100 lines.
So, there is not much to maintain and the code to automatically update the version of the `prettier` is already there.

Even in the worst case scenario, this fork will stop updating, we will have some prettier version in this repo.
It's already working great for lots of files like json/yaml/toml/markdown.
I don't think these files have that much new syntax features added to them (unlike python/c++ code, where you really want tools to work with newer code as well).
So, it's unlikely we'll encounter issues due to not using the latest version.

And the sooner we add prettier, the less diff this PR will generate.